.ci/aarch64/install_rom_aarch64.sh

```diff
@@ -112,8 +112,7 @@ install_uefi_flash()
 clean_up_and_die()
 {
 	clean_up
-	echo "ERROR: $*" >&2
-	exit 1
+	die "$*"
```

Comment:
Hi @jodh-intel , don't you need to source `lib/common.bash` in this script?

Reply:
Good catch - fixed.